syntaxalert.py

```diff
@@ -31,6 +32,17 @@ def loadWord(filename):
 # it holds the files name passed and the stat os file
 files = {}
 
+wrong_word_counter = {}
+
+keyboard.start_recording()
+
+def mispell_callback():
+    recorded_words = keyboard.stop_recording()
+    recorded_words_list = list(keyboard.get_typed_strings(recorded_words))
+    wrong_word = recorded_words_list[0].split()[-1]
```

Comment:
I got: 
```
  File "/usr/local/lib/python3.6/dist-packages/keyboard/_generic.py", line 22, in invoke_handlers
    if handler(event):
  File "/usr/local/lib/python3.6/dist-packages/keyboard/__init__.py", line 1106, in handler
    callback()
  File "/home/mte90/Desktop/Prog/syntaxalert/syntaxalert.py", line 42, in mispell_callback
    wrong_word = recorded_words_list[0].split()[-1]
IndexError: list index out of range
```

Reply:
Exception is thrown every time you digit something or only some times ?

Reply:
On the first word fixed I got that.